server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/eml/embeddedMultipartWithInlineTextAttachment.eml

```diff
@@ -0,0 +1,41 @@
+Date: Tue, 03 Jan 2017 16:05:01 +0100
+From: sender <sender@james.org>
+MIME-Version: 1.0
+To: David DOLCIMASCOLO <david.ddo@linagora.com>
```

Comment:
Anonymizing Benoit's style. You can check that I've obfuscated the mail address!